### DISCUSSION TITLE
Preferred way to install podman-compose (RHEL9)

### DISCUSSION THREAD

**Discussion by Verhoeckx** [containers/podman-compose#618] [General]
The readme.md file advices to install podman-compose with pip3 but is `sudo dnf install podman-compose` (RHEL9) not much easier? Is this not the preferred way to install podman-compose?

**Comment by gbakeman** [comment]
It's how I installed it, although I'm running into some old bugs so using pip to get the dev version may end up being necessary until a release is made.
